CVS shred.c has been rewritten so this code is irrelevant

--- fileutils-4.0u/src/shred.c
+++ fileutils-4.0u/src/shred.c
@@ -853,7 +853,7 @@
     hrtime_t t = gethrtime ();
     ISAAC_SEED (s, t);
 #else
-# if HAVE_CLOCK_GETTIME		/* POSIX ns-resolution */
+# if HAVE_CLOCK_GETTIME && defined(CLOCK_REALTIME) /* POSIX ns-resolution */
     struct timespec t;
     clock_gettime (CLOCK_REALTIME, &t);
 # else
